sky/serve/__init__.py

```diff
@@ -0,0 +1,27 @@
+"""Modules for SkyServe services."""
+import os
+
+from sky.serve.constants import CONTROLLER_FILE_LOCK_PATH
+from sky.serve.constants import CONTROLLER_IDLE_MINUTES_TO_AUTOSTOP
+from sky.serve.constants import CONTROLLER_PREFIX
+from sky.serve.constants import CONTROLLER_RESOURCES
+from sky.serve.constants import CONTROLLER_TEMPLATE
+from sky.serve.constants import ENDPOINT_PROBE_INTERVAL
+from sky.serve.constants import LB_CONTROLLER_SYNC_INTERVAL
+from sky.serve.constants import LOAD_BALANCER_PORT_RANGE
+from sky.serve.constants import SERVE_PREFIX
```

Comment:
Naming: how about SKYSERVE_METADATA_DIR or SKYSERVE_STATE_DIR?

Reply:
Changed to `SKYSERVE_METADATA_DIR`. Thanks!